libs/domain/picking/src/components/picking/picking.component.ts

```diff
@@ -133,7 +134,28 @@ export class PickingComponent extends PickingListMixin(LitElement) {
   }
 
   protected renderFallback(): TemplateResult {
-    return html`<p>${i18n('picking.no-picking-items-found')}</p>`;
+    const pickedCount = this.pickingList?.items.filter(
+      (item) => item.status === ItemsFilters.Picked
+    );
+
+    const isPickingComplete =
+      pickedCount?.length === this.pickingList?.items.length;
+
+    return html`
+      ${when(
```

Comment:
Should we still also display something in case there are no picking items and they aren't complete?

Reply:
As previous behaviour on PA, there was no message or content on this case. When picking are complete, it will appear this success placeholder on tabs with no picking items